diff --git a/dist/client/__vitest_browser__/tester-BJtsJFpD.js b/dist/client/__vitest_browser__/tester-BJtsJFpD.js
index 0594ada..d8e8328 100644
--- a/dist/client/__vitest_browser__/tester-BJtsJFpD.js
+++ b/dist/client/__vitest_browser__/tester-BJtsJFpD.js
@@ -2098,6 +2098,9 @@ channel.addEventListener("message", async (e) => {
   if (!("iframeId" in data) || data.iframeId !== getBrowserState().iframeId) {
     return;
   }
+  if (data.event === "ready" || data.event.startsWith("response:")) {
+    return;
+  }
   switch (data.event) {
     case "execute": {
       const { method, files, context } = data;
