diff --git a/LitDialog.js b/LitDialog.js
index f1e613363bff17d230928d0bdd86425750e1305a..862a089294058cfdd1b23338cb9b06bb8505fff0 100644
--- a/LitDialog.js
+++ b/LitDialog.js
@@ -1,4 +1,4 @@
-import { html, render, TemplateResult } from 'lit';
+import { html, render } from 'lit';
 import { classMap } from 'lit/directives/class-map.js';
 
 /**
@@ -161,7 +161,6 @@ export class LitDialog extends HTMLElement {
      * @param {MouseEvent} e 
      */
     _onModalClicked(e) {
-        console.log('Click target is #modal', e.target == this.modal);
         if (e.target==this.modal) {
             this.onCancel(e);
         }
